scenarioo-client/package.json

```diff
@@ -1,76 +1,83 @@
-        "scenarioo-js": "3.0.x",
-        "style-loader": "^0.18.2",
-        "url-loader": "^0.5.9",
-        "webpack": "2.5.0",
-        "webpack-dev-server": "2.4.5",
-        "webpack-merge": "4.1.0"
-    },
-    "engines": {
-        "node": ">=4.4.0"
-    }
+  "name": "scenarioo-client",
+  "description": "Automated Documentation with UI Tests",
+  "repository": "https://github.com/scenarioo/scenarioo.git",
+  "version": "0.0.0",
+  "license": "GPL-3.0",
+  "scripts": {
+    "postinstall": "bower install",
+    "test": "karma start --single-run",
+    "test-watch": "karma start --auto-watch --no-single-run",
+    "webdriver-manager": "webdriver-manager",
+    "pretest-e2e": "webdriver-manager update",
+    "test-e2e": "protractor protractor-e2e.conf.js ",
+    "pretest-e2e-scenarioo": "webdriver-manager update",
+    "test-e2e-scenarioo": "protractor protractor-e2e-scenarioo.conf.js",
+    "protractor": "protractor",
+    "prebuild": "npm run lint",
+    "build": "webpack --config configs/webpack.config.prod.js",
+    "webpack": "webpack",
+    "webpack-dev-server": "webpack-dev-server",
+    "karma": "karma",
+    "start": "webpack-dev-server -d --inline --progress --port 8500",
+    "eslint": "eslint",
+    "lint": "eslint \"app/**/*.js\" \"test/**/*.js\" --ignore-pattern app/components"
+  },
+  "dependencies": {
+    "angular": "1.5.2",
+    "angular-i18n": "1.5.2",
+    "angular-local-storage": "^0.2.7",
+    "angular-resource": "1.5.2",
+    "angular-route": "1.5.2",
+    "angular-sanitize": "1.5.2",
+    "angular-ui-bootstrap": "1.2.2",
+    "bootstrap": "^3.3.7",
+    "jquery": "3.2.1",
+    "svg-pan-zoom": "github:ariutta/svg-pan-zoom#3.2.11",
+    "twigs": "github:hatchteam/twigs#0.2.3",
+    "typescript": "2.6.1"
+  },
+  "devDependencies": {
+    "@types/angular": "1.6.39",
+    "@types/angular-mocks": "1.5.11",
+    "@types/jasmine": "^2.8.2",
+    "@types/jasminewd2": "^2.0.3",
+    "angular-mocks": "1.5.2",
+    "bower": "1.7.x",
+    "copy-webpack-plugin": "4.1.1",
+    "css-loader": "0.28.1",
+    "eslint": "4.10.0",
+    "file-loader": "0.11.1",
+    "gaze": "1.0.x",
+    "git-rev-sync": "1.6.x",
+    "html-webpack-plugin": "2.28.0",
+    "jasmine-core": "2.4.x",
+    "jasmine-reporters": "2.1.x",
+    "jshint": "2.9.x",
+    "karma": "1.1.x",
+    "karma-jasmine": "1.0.x",
+    "karma-junit-reporter": "1.0.x",
+    "karma-phantomjs-launcher": "1.0.x",
+    "karma-webpack": "^2.0.4",
+    "less": "^2.7.3",
+    "less-loader": "^4.0.5",
+    "lodash": "4.12.x",
+    "moment": "2.13.x",
+    "null-loader": "^0.1.1",
+    "phantomjs-prebuilt": "^2.1.16",
+    "protractor": "^5.2.0",
+    "raw-loader": "0.5.1",
+    "requirejs": "2.2.x",
+    "scenarioo-js": "3.0.x",
+    "style-loader": "^0.18.2",
+    "ts-loader": "3.1.1",
+    "ts-node": "3.3.0",
+    "url-loader": "^0.5.9",
+    "webpack": "2.5.0",
+    "webpack-dev-server": "2.4.5",
+    "webpack-merge": "4.1.0"
+  },
+  "engines": {
+    "node": ">=4.4.0"
+  }
```

Comment:
@danielsuter please revert to old formatting - this change makes it not possible to see what really changed!
Seems like your formatting settings are wrong ?